wiki/Medals/Unlock_requirements/Hush-hush/en.md

```diff
@@ -0,0 +1,8 @@
+# Unlock requirements for Hush-Hush medals
+
+*Note: While some conditions for Hush-Hush medals are known, their solutions are intentionally omitted from this wiki to help preserve their secrecy.*
+
+- A medal only appears in the profile of [gamemodes](/wiki/Game_mode) it can be obtained in. However, there are exceptions due to various reasons, such as star rating changes.
```

Comment:
what do star rating changes have to do with which game mode a medal is available on?

Reply:
> what do star rating changes have to do with which game mode a medal is available on?

There are multiple medals that have star rating requirement (for example need to be played on 4*+), but in specific gamemodes such high star rating may not exist even with mods applied, making it impossible to get in said gamemode.

Reply:
this doesn't sound like a real thing, likely some kind of coincidence?
```suggestion
- A medal only appears in the profile of [gamemodes](/wiki/Game_mode) it can be obtained in.
```

Reply:
> this doesn't sound like a real thing, likely some kind of coincidence?

I mean, yea, I believe it's not intentional, but that happens relatively often, when all 4 gamemodes are not considered properly. Also reminds me of somewhat recent incident with Not Bluffing not working (Ephemeral fixed it later) after osu! star rating changes.

Reply:
i don't think this should account for unexpected/unintentional bugged cases, best to keep it straightforward.